__tests__/app.test.js

```diff
@@ -142,7 +142,207 @@ describe('4. GET /api/articles/:article_id', ()=>{
+        const newComment = {
+            username: "butter_bridge",
+            body: "my eyes hurt from looking at this article"
+        };
+        return request(app)
+        .post('/api/articles/6/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body})=>{
+            expect(body.comment[0]).toEqual({
+                comment_id: 19,
+                body: "my eyes hurt from looking at this article",
+                article_id: 6,
+                author: "butter_bridge",
+                votes: 0,
+                created_at: expect.any(String)
+            })
+        });
+    });
+    test('404: posting a comment to a valid but non-existent article returns not found', ()=>{
+        const newComment = {
+            username: "butter_bridge",
+            body: "my eyes hurt from looking at this article"
+        };
+        return request(app)
+        .post('/api/articles/100/comments')
+        .send(newComment)
+        .expect(404)
+        .then((response)=>{
+            const msg = response.body.msg;
+            expect(msg).toBe('Not Found');
+        });
+    });
+    test('400: posting a comment to a valid article_id with a null key (that has a NOT NULL constraint) returns bad request', ()=>{
+        const newComment = {
+            username: "",
+            body: "the grass is greener on the other side :)"
+        };
+        return request(app)
+        .post('/api/articles/10/comments')
+        .send(newComment)
+        .expect(400)
+        .then((response)=>{
+            const msg = response.body.msg;
+            expect(msg).toBe('Bad Request');
+        });
+    });
+    test('400: posting a comment to a valid article_id with a missing key (that has a NOT NULL constraint) returns bad request', ()=>{
+        const newComment = {
+            username: "icellusedkars",
+        };
+        return request(app)
+        .post('/api/articles/10/comments')
+        .send(newComment)
+        .expect(400)
+        .then((response)=>{
+            const msg = response.body.msg;
+            expect(msg).toBe('Bad Request');
+        });
+    });
+    test('400: posting a comment to an invalid article id returns bad request', ()=>{
+        const newComment = {
+            username: "icellusedkars",
+            body: "my eyes hurt from looking at this article"
+        };
+        return request(app)
+        .post('/api/articles/DROPDATABASE/comments')
+        .send(newComment)
+        .expect(400)
+        .then((response)=>{
+            const msg = response.body.msg;
+            expect(msg).toBe('Bad Request');
+        })
+    });
+});
+
+describe('7. PATCH api/articles/:article_id', () => {
+    test('responds with the newly updated article after vote increment', () => {
+      return request(app)
+        .patch('/api/articles/3')
+        .send({ inc_votes: 45 })
+        .expect(200)
+        .then(({body})=>{
+          expect(body.article).toMatchObject([{
```

Comment:
Your article property should be an object rather than an array containing a single object